md/tomcat-bundle.md

```diff
@@ -106,30 +106,38 @@ For production, you are recommended to use one of the supported databases, with
 
-If you use the [Business Data Model (BDM) feature](define-and-deploy-the-bdm.md), we recommend that you configure a dedicated database.  
+If you use the [Business Data Model (BDM) feature](define-and-deploy-the-bdm.md), we recommend that you configure a dedicated database.
 You can also find more details on database configuration in the [dedicated page](database-configuration.md).
 
-Follow those steps:
+Follow those steps (see [Bundle auto-configuration](BonitaBPM_platform_setup.md#run_bundle_configure) for advanced usage of the configure command):
+
+1. Edit file `[TOMCAT_HOME]`/setup/**database.properties** and modify the properties to suit your databases (Bonita BPM internal database & Business Data database).
+2. If you use **Microsoft SQL Server** or **Oracle database**, copy your database drivers in `[TOMCAT_HOME]`/setup/lib folder. (Open-source drivers for H2, MySQL and PostgreSQL are already shipped in the tool)
+3. Run `[TOMCAT_HOME]`/setup/**setup.sh configure** (Unix system) or `[TOMCAT_HOME]`\setup\setup.bat configure (Windows system).
 
-1. Drop your database vendor-specific drivers in `[TOMCAT_HOME]`/lib/bonita (you can copy the provided open-source drivers: PostgreSQL, MySQL) from `[TOMCAT_HOME]/setup/lib`
-2. Edit file `[TOMCAT_HOME]`/conf/ **bitronix-resources.properties**
-    1. Comment the default embedded h2 database configuration (preceding the lines with a #)
-    2. Uncomment the configuration for your database vendor (PostgreSQL, Oracle, SQL Server, or MySQL)
-    3. Change the default values for your database configuration to point to an existing database instance and valid credentials
-  Warning: this must be done for 2 different datasources in the file: **resource.ds1.** (for engine and configuration data) and **resource.ds2.** (for BDM data, optional but handy to configure in case some day you need to use it)
-3. Edit file `[TOMCAT_HOME]`/conf/Catalina/localhost/ **bonita.xml**
-    1. Comment the default embedded H2 database configuration (with `<!--` and `-->` around the lines to comment)
-    2. Uncomment the configuration for your database vendor (PostgreSQL, Oracle, SQL Server, or MySQL)
-    3. Change the default values for your database configuration to point to an existing database instance and valid credentials
-  Warning: this must be done for 2 different datasources in the file: **bonitaSequenceManagerDS** (for engine and configuration data, same base as **resource.ds1.**) and **NotManagedBizDataDS** (for BDM data, same base as **resource.ds2.**)
-4. Edit file sentenv.sh (Unix system) or setenv.bat (Windows system)
-    1. For engine and configuration data, change the **DB_OPTS** property and change the default **h2** value for the one corresponding to your database vendor 
-    2. For BDM data, change the **BDM_DB_OPTS** property and change the default **h2** value for the one corresponding to your database vendor 
 
+::: info
+What is the **bonita-start.sh** script doing?
```

Comment:
We're thinking of rewriting it with 2 sections:
* Install it in a nutshell for beginners who don't want to tune everything.
* Advanced installation with more details.